aem-authoring-toolkit-samples/toolkit-samples-package/src/main/content/META-INF/vault/filter.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<workspaceFilter version="1.0">
+    <filter root="/apps/authoring-toolkit/samples"/>
+    <filter root="/apps/acs-list"/>
+    <filter root="/content/aem-authoring-toolkit-samples"/>
+    <filter root="/content/acs-list"/>
+    <filter root="/content/cq:tags"/>
+    <filter root="/etc/designs"/>
```

Comment:
these filters are too general and can potentially delete already existing user content

Reply:
Also it looks like not all of them are needed (like acs-list content, etc.)